tsconfig.spec.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "./tsconfig.json",
-  "include": ["./jest/testing-library__jest-dom.d.ts", "./jest/verifyComponent.d.ts", "**/*.spec.ts?(x)"],
+  "include": ["./jest/testing-library__jest-dom.d.ts", "./jest/verifyComponent.d.ts", "**/*.spec.tsx"],
```

Comment:
This fixes the issue where spec files weren't being properly type checked in VSCode. `verifyComponent` was registering as undefined and `expect` was returning `any`